packages/shared-state-async/files/usr/libexec/rpcd/shared-state-async

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env lua
+
+--[[
+Shared State Async
+
+Copyright (c) 2024  Javier Jorge <jjorge@inti.gob.ar>
+Copyright (c) 2024  Instituto Nacional de Tecnolog..a Industrial
+Copyright (C) 2024  Asociacion Civil Altermundi <info@altermundi.net>
+
+This is free software, licensed under the GNU AFFERO GENERAL PUBLIC LICENSE Version 3
+]]
+--
+local utils = require('lime.utils')
+local json = require 'luci.jsonc'
+
+local function get(msg)
+    local error = os.execute("shared-state-async get " ..
+        msg.data_type .. " 2>/dev/null ")
+    utils.printJson({
```

Comment:
That the error get printed only on failure happens implicitely, please add at least a comment documentig this behaviour, so we cna understand what's happening when we look at this code in a few months or years ;)

Reply:
Done ! it is a weird behavior that gives a lot of headaches. Any output that is printed in the console is used as output. If its a json everything goes fine, if it is not a json you get "invalid input" as a response.